test/bdd/vdr/orb/create_follow_activity.sh

```diff
@@ -24,7 +24,13 @@ echo "$OSTYPE"
 if [ "$OSTYPE" == "darwin20" ]; then
 ./orb-cli-darwin-amd64 follower --outbox-url=https://localhost:8009/services/orb/outbox --actor=$domain2IRI --to=$domain1IRI --action=Follow --tls-cacerts=../fixtures/keys/tls/ec-cacert.pem --auth-token=ADMIN_TOKEN
 ./orb-cli-darwin-amd64 witness --outbox-url=https://testnet.orb.local/services/orb/outbox --actor=$domain1IRI --to=$domain2IRI --action=InviteWitness --tls-cacerts=../fixtures/keys/tls/ec-cacert.pem --auth-token=ADMIN_TOKEN
+./orb-cli-darwin-amd64 ipfs key-gen --ipfs-url=http://localhost:5001 --key-name=key1 --privatekey-ed25519=6dFbUloT+39dRpeCAu8gVYNJu3CXD/6SJsBGSSpe0k4ON1FS5idB/HHMkjYAsrBXm+34xT1NTGFTPX3mZFsiLA || true
```

Comment:
Suggest setting a variable with the binary rather than repeating every execution line.

Reply:
done